eng/copy-wpf.ps1

```diff
@@ -0,0 +1,126 @@
+# Copy wpf binaries from local build to desired location. The location can either be the version number (for copying to shared installation)
+# or the location of the project file if copying binaries to the output of a locally built application.
+Param(
+[string]$destination,
+[string]$arch="x86",
+[switch]$release,
+[switch]$shared,
+[switch]$help
+)
+
+function CopyByBinaryToPublishLocation($publishLocation)
+{
+    $locallyBuiltBinaryLocationBase = Join-Path $RepoRoot "artifacts\bin"
+    CopyNativeBinariesToPublishLocation $publishLocation $locallyBuiltBinaryLocationBase 
+    CopyManagedBinariesToPublishLocation $publishLocation  $locallyBuiltBinaryLocationBase
+    CopyThemeBinariesToPublishLocation $publishLocation  $locallyBuiltBinaryLocationBase
+    CopyUIAutomationBinariesToPublishLocation $publishLocation  $locallyBuiltBinaryLocationBase
+}
+
+function CopyNativeBinariesToPublishLocation($publishLocation, $localBinLocation)
+{
+    $NativeBinaries = "D3DCompiler", "PenImc", "PresentationNative", "wpfgfx"
```

Comment:
We also depend on vcruntime140_cor3. This dependency will light-up as soon as the managed portion of /dllrename functionality in the linker starts working.

Reply:
i see vcruntime140d.dll in my artifacts\bin directory right now, is this something that's going to come later?

Reply:
now that we are copying by package, i think this is no longer relevant